app/data_models/answer.py

```diff
@@ -21,14 +21,20 @@
 ]
 
 
+class AnswersDictType(TypedDict, total=False):
```

Comment:
Singular, as it represents 1 answer. I also don't think we need `Type` suffix for these one as it can be used as a normal dict, like list store does.

Reply:
Fixed now, thanks!